packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -26,6 +26,8 @@ $inputBackgroundCheckedColor: "[theme:inputBackgroundChecked, default: #0078d7]"
 $inputBackgroundCheckedHoveredColor: "[theme:inputBackgroundCheckedHovered, default: #106ebe]";
 $inputForegroundCheckedColor: "[theme:inputForegroundChecked, default: #ffffff]";
 $inputFocusBorderAltColor: "[theme:inputFocusBorderAlt, default: #0078d7]";
+$smallInputBorderColor: "[theme:inputBorder, default: #666666]";
+$smallInputBorderHoveredColor: "[theme:inputBorderHovered, default: #212121]";
```

Comment:
Can we reuse inputBorderHovered for now until we know we need this new smallInputBorderHovered?

Reply:
Done.